lib/kitchen/directions/bake_example.rb

```diff
@@ -20,21 +20,14 @@ def self.v1(example:, number:, title_tag:, numbered_solutions: false)
                .pantry(name: :link_text)
                .store("#{I18n.t(:example_label)} #{number}", label: example.id)
 
-        example.titles.each do |title|
-          if title.parent.has_class?('os-caption-container') || \
-             title.parent.has_class?('os-caption') || \
-             title.parent.name == 'caption'
-            next
-          end
-
+        example.titles_to_rename.each do |title|
           title.name = 'h4'
         end
 
         example.exercises.each do |exercise|
           next if exercise.baked?
 
           if (problem = exercise.problem)
-            problem.titles.each { |title| title.name = 'h4' }
```

Comment:
removed this because an exercise within an example should already have its title converted to an H4 by the above, & tables within exercises within examples were being mistakenly impacted